src/Legacy/DynamoRevitVersionSelector/Properties/Resources.en-US.resx

```diff
@@ -133,4 +133,7 @@
   <data name="DynamoVersionTooltip" xml:space="preserve">
     <value>Launches Dynamo {0}, to launch a different version of Dynamo you may need to restart Revit</value>
   </data>
+  <data name="VisualProgramming" xml:space="preserve">
```

Comment:
App_Description is defined in DynamoRevit project, and that dll doesn't get loaded until DynamoRevitVersionSelector is loaded. Also, DynamoRevit shouldn't depend on version selector, so there we can't used resource defined here.